innsender/src/main/kotlin/no/nav/soknad/innsending/repository/HendelseType.kt

```diff
@@ -0,0 +1,13 @@
+package no.nav.soknad.innsending.repository
+
+enum class HendelseType {
+	Ukjent,
+	Opprettet,
```

Comment:
Legg på `endret` for mellomlagring når det er endringer på søknaden (PUT i fyllUt-api)